build.gradle

```diff
@@ -738,6 +738,13 @@ project(':iceberg-arrow') {
       exclude group: 'io.netty', module: 'netty-common'
       exclude group: 'com.google.code.findbugs', module: 'jsr305'
     }
+
+    testCompile("org.apache.arrow:arrow-memory-netty") {
```

Comment:
these dependencies are needed for the ArrowReaderTest.

Reply:
how come `arrow-memory-netty` is listed twice?

Reply:
I will remove testCompile instance.